### PR TITLE
adds NLP endoscopy project to work

### PR DESCRIPTION
**Pull request by laurencejackson**

### CODE REVIEW

File: _data/work.yml [hunk 1]
```diff
@@ -88,4 +88,11 @@
   plan: current
   dataset: done
   training:
+  test:
+
+- title: Natural Language Processing of Endoscopic and associated pathology text to optimise post COVID endoscopy resources
+  link: projects/AIC016-NLP_endoscopic_resources_optimisation.html
```
Comment (hshuaib90) [inline _data/work.yml]: 16 is already taken according to https://docs.google.com/spreadsheets/d/1mII4MGBEyfO7FEuJPtCA0AbMIAiuC8sFZLaH64Mpvj4/edit#gid=1453792661. It seems @AnilMistry entry is wrong. @laurencejackson  can you edit his stuffy to AIC016 and change yours to AIC017.

**Review by hshuaib90 — CHANGES_REQUESTED** [review]
Can you also please remove the .DS_Store from the commit.

File: _data/work.yml (hunk 1 above)
Comment (laurencejackson) [reply _data/work.yml]: Should be fixed now, I've not renamed the branch so we don't need to open another PR, but the content now only refers to this project as AIC017.

File: _data/work.yml [hunk 2]
```diff
@@ -88,4 +88,11 @@
   plan: current
   dataset: done
   training:
+  test:
+
+- title: Natural Language Processing of Endoscopic and associated pathology text to optimise post COVID endoscopy resources
+  link: projects/AIC017-NLP_endoscopic_resources_optimisation.html
```
Comment (hshuaib90) [inline _data/work.yml:94]: Doesn't look like you've merged latest master as ANil's project still has 018

**Review by hshuaib90 — CHANGES_REQUESTED** [review]

File: _data/work.yml (hunk 2 above)
Comment (laurencejackson) [reply _data/work.yml:94]: should be resolved now

**Review by hshuaib90 — APPROVED** [review]

**Pull request merged**